test_util/marathon.py

```diff
@@ -9,7 +9,17 @@
 
 import test_util.helpers
 
+DEFAULT_API_BASE = '/marathon/v2'
```

Comment:
Should there not be a default api version? I just figured the pods tests can then specifically ask for a marathon api with a different version